dask/array/core.py

```diff
@@ -1047,6 +1047,19 @@ def _set_chunks(self, chunks):
     def __len__(self):
         return sum(self.chunks[0])
 
+    def __array_ufunc__(self, numpy_ufunc, method, *inputs, **kwargs):
+        import pdb; pdb.set_trace()
+        from . import ufunc
+        da_ufunc = getattr(ufunc, numpy_ufunc.__name__)
+        if method == '__call__':
+            return da_ufunc(*inputs, **kwargs)
+        elif method == 'outer':
+            return da_ufunc.outer(*inputs, **kwargs)
+        else:
+            from ..base import compute
+            inputs2 = compute(*inputs)
```

Comment:
What would you recommend?  I've changed this to returning `NotImplemented` for now.